src/types.ts

```diff
@@ -11,6 +11,8 @@ import type { ICustomType } from "./types/types.ts";
 // An app manifest is generated based on what this has defined in it
 
 export type {
+  ActionContext,
+  ActionHandler,
```

Comment:
Yeah that's a good point. More of a leftover from initial testing; since the initial action registering `add` method didn't support complex matching to action IDs, I had to register it multiple times - thus why I assigned it to a variable. I imagine once I implement the more complex matching that Brad suggests, in practice this wouldn't be needed.

I will tackle the more complex action matching that Brad suggests first, and then based on that, remove exporting any of the underlying types if it's not needed.